Source/GTMSessionFetcher.m

```diff
@@ -819,11 +819,12 @@ - (void)beginFetchMayDelay:(BOOL)mayDelay
 #endif  // GTM_DISABLE_FETCHER_TEST_BLOCK
 
 #if GTM_BACKGROUND_TASK_FETCHING
-  // Background tasks seem to interfere with out-of-process uploads and downloads.
-  if (!self.skipBackgroundTask && !self.useBackgroundSession) {
+  // Background tasks seem to interfere with out-of-process uploads and downloads. Background
```

Comment:
I reverted this part. I'm not familiar enough with how GTM uses background sessions (as opposed to UIApplication background tasks) to really update the comment here if something else is out of date.

I tried to amend my git change and push, but it seems to be having issues. I'm not very familiar with GitHub so it might take me a bit to get it to work properly.